### DISCUSSION TITLE
How to use custom attribute selector

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [tailwindlabs/tailwindcss#10762] [Help] [answered]
Hi, 

I'm trying to apply styles to an element on hover only if it has the `href` selector present (I reuse the same styles for non-link elements as well).

I tried something like that but it doesn't work:
`hover:[[href]]:text-gray-700`

Any help will be appreciated. Thanks

**Comment by wongjn** [comment, marked as answer]
You're missing the ampersand, `hover:[&[href]]:text-gray-700`. [Tailwind Play live example](https://play.tailwindcss.com/LfEsMbXsF1)

**Reply by piotrkulpinski** [reply]
Thank you!
